File: app/src/main/java/com/bottle/track/map/model/TrekPoi.java
package com.bottle.track.map.model;

/**
 * @Date 2017/12/5 15:03
 * @Author halfbottle
 * @Version 1.0.0
 * @Description Trek 中的POI，继承自GeoPoint，即包含坐标位置信息，自定义其它属性，如描述，地名等
 */
public class TrekPoi extends GeoPoint {

    public long logtime;         // 记录时间
    public String province;      // 省
    public String city;          // 市
    public String district;      // 区/县
    public String name;          // 地名
    public String description;   // 描述，提供给用户，作游记等

    public TrekPoi(double longitude, double latitude, double altitude) {
        super(longitude, latitude, altitude);
    }

}

File: app/src/main/java/com/bottle/track/db/table/TrekPoi.kt
package com.bottle.track.db.table

/**
 * @ClassName TrekPoi
 * @Author half_bottle
 * @Date 2017/12/3
 * @Description eTrek 兴趣点的本地数据表
 */
class TrekPoi {

    val longitude: Double             // 经度
    val latitude: Double              // 纬度
    val altitude: Double              // 海拔
    var logtime: Long? = null         // 记录时间
    var province: String? = null      // 省
    var city: String? = null          // 市
    var district: String? = null      // 区/县
    var name: String? = null          // 地名
    var description: String? = null   // 描述，提供给用户，作游记等

    constructor(longitude: Double, latitude: Double, altitude: Double){
        this.longitude = longitude
        this.latitude = latitude
        this.altitude = altitude
    }
}